feature/gnoi/os/tests/osinstall/README.md

```diff
@@ -2,78 +2,96 @@
 
 ## Summary
 
-Validate new software can be copied and activated on single and dual
-supervisors.
+*   Validate new software can be copied and activated on single and dual supervisors.
+*   Validate successful configiuration push to the DUT and also confirm the health of the software-module that allows configuration of the router.
 
-## Procedure
-
-*   Install and activate a new software version on primary supervisor:
-
-    *   Issue gnoi.os.Install rpc to the chassis with
-        InstallRequest.TransferRequest message. The message should set the
-        version to the desired new version image, and standby_supervisor to
-        FALSE.
-
-        *   Wait for the switch to respond with InstallResponse. Expect it to
-            return TransferReady.
-
-    *   Transfer the content by issuing gnoi.os.Install rpc with
-        InstallRequest.transfer_content message.
-
-        *   Expect it to return InstallResponse with a TransferProgress status
-            asynchronously at certain intervals.
-        *   TODO: When the expected amount of bytes_received is reported by the
-            switch, move to the activation step next.
-
-    *   End the transfer of software by issuing gnoi.os.Install rpc with
-        InstallRequest.TransferEnd message.
 
-        *   Expect the switch to return InstallResponse with a Validated
-            message. The version in the message should be set to the one which
-            was transferred above.
-
-    *   Activate the software by issuing gnoi.os.Activate rpc.
-
-        *   Set the version field of the ActivateRequest message to be the same
-            as the version specified in the TransferRequest message above.
-        *   Set the no_reboot flag to true.
-        *   Set the standby_supervisor to FALSE.
-
-*   Install and activate the same new software version on standby supervisor:
-
-    *   Repeat the above process of TransferRequest. This time set the
-        standby_supervisor to TRUE.
-        *   Expect the switch to return a InstallResponse with a SyncProgress
-            message. The switch should sync the software image from primary SUP
-            to standby.
-        *   Expect the sync to return a value of 100 for percentage_transferred
-            field.
-        *   At the end, expect the switch to return InstallResponse with a
-            Validated message. The version in the message should be set to the
-            one which was transferred above.
-
-*   Activate the software by issuing gnoi.os.Activate rpc as in the case of
-    primary supervisor.
-
-    *   Set the version field of the ActivateRequest message to be the same as
-        the version specified in the TransferRequest message above.
-    *   Set the no_reboot flag to true.
-    *   Set the standby_supervisor to TRUE this time.
+## Procedure
 
-*   Reboot the switch:
+* Subtest-1 : OS Install on the primary and secondary controller cards. 
```

Comment:
Subtest-1, 2 so on is the process being followed in other tests too. Also, the test numbering is a little confusing right now. Check for example https://source.corp.google.com/piper///depot/google3/net/management/wbb/catalog/testregistry.textproto;l=40-51. BMP-2.1, 2.2, 2.3, 2.4 ... are separate tests rather than being subtests.

Reply:
subtests should be 

BMP-2.1.1 or such not just subtest-1 that doesn't hold any meaning

Reply:
You mean gNOI-4.1.1 and gNOI-4.1.2? I updated so, ptal